src/components/Buttons/Button/Button.jsx

```diff
@@ -1,23 +1,38 @@
+import { Button as BootstrapButton } from 'react-bootstrap';
 import PropTypes from 'prop-types';
-import noop from '../../../shared/noop';
 import style from './Button.module.css';
+import noop from '../../../shared/noop';
+
+export function Button({ title, onClick, stylingType, isBackButton, history, ...restProps }) {
+  const buttonBackHandler = () => {
+    const { goBack } = history;
+    goBack();
+  };
 
-export function Button({ title, onClick, stylingType, ...restProps }) {
   return (
-    <button className={style[stylingType]} type='button' onClick={onClick} {...restProps}>
-      {title}
-    </button>
+    <BootstrapButton
+      className={style.button}
+      variant={stylingType}
+      type='button'
```

Comment:
add type to props with default value in case you decide to change it

Reply:
```suggestion
      type={buttonType}
      
      buttonType: 'button',
```